packages/trpc/server/routers/viewer/teams/getTeamWithMinimalData.schema.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+export const ZGetTeamWithMinimalDataSchema = z.object({
+  teamId: z.number(),
+  isOrg: z.boolean().optional(),
```

Comment:
Followup: I think it is a bad idea to take an input that we can quickly derive at backend. It could potentially cause inconsistencies in system

We can always determine `isOrg` value from teamId at backend.

Reply:
IMO this is not for a follow up. This should be blocking.